src/index.js

```diff
@@ -1,42 +1,63 @@
+import TodoTask from './modules/todoTask.js';
+import LocalStorage from './modules/local_storage.js';
+import Task from './modules/task.js';
 import './style.css';
 
-let todoList = [
-  {
-    description: 'Giving Feedback',
-    isCompleted: false,
-    index: 3,
-  },
-  {
-    description: 'Going on field',
-    isCompleted: false,
-    index: 2,
-  },
-  {
-    description: 'Tasks dispatching',
-    isCompleted: false,
-    index: 1,
-  },
-  {
-    description: 'Meeting',
-    isCompleted: false,
-    index: 0,
-  },
-];
-
-localStorage.setItem('todo', JSON.stringify(todoList));
-
-const todoContainer = document.querySelector('#todo');
-const populateList = () => {
-  todoList = JSON.parse(localStorage.getItem('todo'));
-  todoList.forEach((item) => {
-    const li = document.createElement('li');
-    li.innerHTML = `<button class="check-item">
-            <i class="fa-regular fa-square">
-            </i>${item.description}</button>
-            <button class="move-item">
-            <i class="fa-solid fa-ellipsis-vertical"></i>
-            </button>`;
-    todoContainer.insertBefore(li, todoContainer.children[item.index]);
-  });
+const inputTodo = document.getElementById('input-todo');
+const addTodo = document.querySelector('.add-todo');
+const error = document.querySelector('.error-msg');
+const reload = document.querySelector('.reload');
+// const clearCompleted = document.querySelector('.clear');
```

Comment:
- [ ] Please remove this commented code because it is a bad practice comments should be reserved to explain difficult-to-understand code or the reason for doing something out of the ordinary. Don't worry code should be stored in source control :+1: